lib/commands/deploynodeapp.js

```diff
@@ -7,7 +7,11 @@ var async = require('async');
 var fs = require('fs');
 var mustache = require('mustache');
 var _ = require('underscore');
+var post = require('request').post;
 var unzip = require('node-unzip-2');
+var targz = require('tar.gz')({}, {
```

Comment:
we support `tar, tgz, zip` and i wrote this code before we had zip support as that was a newer addition.